### DISCUSSION TITLE
How to replace a battery powered node which is not marked as dead

### DISCUSSION THREAD

**Discussion by akamming** [zwave-js/zwave-js-ui#3611] [General]
Hi,

In openzwave i had a "is node alive" call, which marked nodes as dead when it could not connect to these nodes.

Currently i have 2 battery powered nodes, which don't reach the dead state:
- One is from migration. This was an already dead node, so it never finished the interview stage. I did not remove it, cause i always "replace dead nodes" if possible to reuse id's (i'm already on 195, so i try not to use to many id's ;-))
- Another is a Heiman smokesensor which just stopped working. It seems to remains in a sleep state forever...

I'd like to mark both as dead, so in the future i can use the "replace failed node" to re-use the id's...   (and of course i already tried this, but that function fails, cause the node is not dead. I can remove the device, but as stated i would really like to reuse the id).

Is there a way to do this in zwave js ui?

**Comment by kpine** [comment]
Go to the control panel, click the hamburger button (lower right blue button), then "Manage nodes". choose "Replace" then select the battery node to replace. No need to mark anything dead, as battery devices cannot be dead. If you had some error, please provide screenshot and debug logs.

**Reply by robertsLando** [reply]
I would also add that the node should be factory resetted before doing this, when doing that btw the controller stick should be unplugged to be sure the node doesn't tell him he has been removed from network

**Reply by kpine** [reply]
Whoops, I misread the question. Yes, the you need to do this for the replace to be effective if the node hasn't been removed already. Or exclude the node using a separate controller.

**Reply by akamming** [reply]
The nodes are alreqdy dead, so cannot exclude them. 

I don’t have a new node yet to replace them, but will try when I have a new device. Tx for the help and keep up the good work!
